subsys/bluetooth/controller/Kconfig

```diff
@@ -524,6 +525,11 @@ config BT_CTLR_ASSERT_HANDLER
 	  and will be invoked whenever the controller code encounters
 	  an unrecoverable error.
 
+config BT_CTLR_DF
```

Comment:
Now `BT_CTLR_DF` is located in the group "BLE Controller debug configuration". That is probably not intentional

Reply:
Fixed. 
Anything that is included or added after Kconfig.ll_sw_split will end in that group unless new group (comment) is added after inclusion of Kconfig.ll_sw_split. 
I suggest to do something with Kconfig.ll_sw_split. It should not affect general controller Kconfig when is included there.